test/compilable/issue16044.d

```diff
@@ -0,0 +1,8 @@
+// REQUIRED_ARGS: -Icompilable/imports
+module issue16044; // https://issues.dlang.org/show_bug.cgi?id=16044
+
+import pkg16044;
+import pkg16044.sub;
+
+static assert([__traits(allMembers, pkg16044)] == ["object", "test1", "test2"]);
```

Comment:
Why is `object` showing up in allMembers?

Reply:
It's always been the case, it's the object.d automatic import... try in a terminal module, e.g std.range.primitives, it will show up too.

Reply:
```d
import std.range.primitives;
import std;
void main()
{
    static assert([__traits(allMembers, std.range.primitives)].canFind("object"));
}
```